packages/wes_adapter/amazon_genomics/wes/adapters/BatchAdapter.py

```diff
@@ -180,7 +183,25 @@ def describe_job(self, job_id: str) -> Optional[JobDetailTypeDef]:
     def describe_jobs(self, job_ids: typing.List[str]) -> typing.List[JobDetailTypeDef]:
         if not job_ids:
             return []
-        return self.aws_batch.describe_jobs(jobs=job_ids)["jobs"]
+
+        jobs = []
+        job_ids_sets = chunks(job_ids, 100)
+        with ThreadPoolExecutor(max_workers=10) as executor:
+            future_jobs = {
+                executor.submit(
+                    self.aws_batch.describe_jobs, jobs=job_ids_set
+                ): job_ids_set
+                for job_ids_set in job_ids_sets
+            }
+            for future in as_completed(future_jobs):
+                job_ids_set = future_jobs[future]
+                try:
+                    response = future.result()
+                    jobs += response["jobs"]
+                except Exception as e:
+                    print(f"error retrieving jobs: {e}")
```

Comment:
Can `self.logger` be used here for consistency with other classes?

Reply:
Yes